tests/kuttl/tests/dataplane-deploy-no-nodes-test/01-assert.yaml

```diff
@@ -398,6 +399,56 @@ status:
 ---
 apiVersion: ansibleee.openstack.org/v1beta1
 kind: OpenStackAnsibleEE
+metadata:
+  name: install-certs-edpm-compute-no-nodes
+  namespace: openstack
+  ownerReferences:
+  - apiVersion: dataplane.openstack.org/v1beta1
+    blockOwnerDeletion: true
+    controller: true
+    kind: OpenStackDataPlaneDeployment
+    name: edpm-compute-no-nodes
+spec:
+  backoffLimit: 6
+  extraMounts:
+  - mounts:
```

Comment:
Won't the service add some volumes of it's own?

Reply:
Only when we do the test with tlsEnabled: true

That's coming -- and you've already been involved in that patch.